vllm/spec_decode/spec_decode_worker.py

```diff
@@ -91,6 +98,9 @@ def create_worker(
         scorer_worker: WorkerBase,
         draft_worker_kwargs: Dict[str, Any],
         disable_by_batch_size: Optional[int],
+        draft_token_sampling_method: Optional[str] = "rejection_sampler",
+        typical_acceptance_sampler_posterior_threshold: Optional[float] = 0.09, 
+        typical_acceptance_sampler_posterior_alpha: Optional[float] = 0.3, 
```

Comment:
let's remove all defaults that aren't in the Config layer. otherwise it gets tricky since we have multiple sources of truth

Reply:
Done.